backend/server/README.md

```diff
@@ -65,12 +40,20 @@ if you have `docker` installed, then
   docker-compose up --build
   ```
 
-The first time it will take time to build and run. Every other time, it will take 2-3 minutes to setting up and 20-25 seconds to shutdown the containers.
+The first time it will take time to build and run. Every other time, it will take 2-3 minutes to set up and 20-25 seconds to shut down the containers.
+
+## Usage 
+
+The API documentation can be accessed via the `\docs` endpoint. 
+
+### Authentication
+
+The API uses [JSON Web Tokens](https://jwt.io/) for authentication and authorization.
 
-## Contributing
+To access private endpoints, you must first authenticate as a user using the `auth\login` endpoint. Upon successful login, the server responds with an access token, which is set as an HTTP-only cookie in the response. The cookie is then automatically attached to all of the requests to the same domain.
 
-Pull requests are welcome! For major changes, please open an issue first to discuss what you would like to change.
+Note that the lifespan of the access token is short-lived. If you need to refresh the token, you can do so via the `auth/refresh` endpoint.
 
-## License
+### Authorization
 
-See `LICENSE` for more information.
+The roles of the users are scoped to tournaments and matches. I.e., the tournament organizers can act as an administrator for their organized tournament. The role of an official needs to be assigned by the tournament administrator.
```

Comment:
Note that this is not the current way we authorize users, but how it will be changed in the near future.